index.html

```diff
@@ -1,21 +1,66 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>Page Title</title>
-    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;1,300&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <div class="container">
-      <header></header>
-      <section class="section-1">
-        <h1>I'm Marine Glad to see you</h1>
-        <p>
-          I'm a software Developer! I can help you build aproduct, feature or
-          website look through some of my work and exprience! if you like what
-          you se & have aproject you need, don't hestiate to contact me
-        </p>
-      </section>
+      <main>
+        <header>
+          <div class="logo">
+            <a href="#">
+              <img
+                src="Asset/My Logo.png"
+                alt="log for My portifolio"
+                class="logo"
+            /></a>
+          </div>
+          <nav>
+            <img src="Asset/hamburger.png" alt="Menu Icon" class="menu-icon" />
+          </nav>
```

Comment:
Great work so far 💪💪, only small things to fix before moving to the next week, this icon needs to be aligned to the right good job on using flexbox correctly ✔ you have just to remove the horizontal margin to get the desired design 
Also don't forget to increase the size of the title to make it look like in the template

![mobile first](https://user-images.githubusercontent.com/59707859/235805640-393ea3cc-91db-4c21-8ddb-06b427316245.PNG)